src/components/Charts/charts.css

```diff
@@ -0,0 +1,4 @@
+svg {
+    background-color: #677680;
+    margin-top: 25px;
+}
```

Comment:
this is going to apply to any svg associated with the website.  This should have a selector on it like a #chart

Reply:
use scss